src/Tempest/Log/src/GenericLogger.php

```diff
@@ -74,14 +74,14 @@ private function writeLog(Level $level, string $message, array $context): void
 
     private function resolveDriver(LogChannel $channel, Level $level): Monolog
     {
-        if (! isset($this->drivers[$channel::class])) {
-            $this->drivers[$channel::class] = new Monolog(
+        if (! isset($this->drivers[spl_object_id($channel)])) {
```

Comment:
why `spl_object_id` and not simply append `$this->drivers[]` ?

Reply:
I think I don't understand? `$drivers` is a map of `LogChannel` to `Monolog` instance, but as we cannot use `LogChannel` as offset, I decided to use its id (as `::class` is not unique).

Do you mean to resolve it once, and just iterate over drivers if already resolved?

Reply:
My bad, I was confused for a second 😅